include/uxr/agent/client/session/Session.hpp

```diff
@@ -187,13 +212,13 @@ inline bool Session::push_output_message(dds::xrce::StreamId stream_id, OutputMe
     if (128 > stream_id)
     {
         std::lock_guard<std::mutex> bo_lock(bo_mtx_);
-        besteffort_ostreams_[stream_id].promote_stream();
+        besteffort_out_streams_.at(stream_id).promote_stream();
         rv = true;
     }
     else
     {
         std::lock_guard<std::mutex> ro_lock(ro_mtx_);
-        rv = relible_ostreams_[stream_id].push_message(output_message);
+        rv = reliable_out_streams_.at(stream_id).push_message(output_message);
```

Comment:
Because it is faster than `operator[]` and I don't need to perform a bound checking because I created the element in the constructor.

Reply:
I was wrong, it is not faster, in fact both operators have the same complexity, but I have to use `at` because `OutputStreams` do not have a default constructor.

Reply:
I have read about this, and both check the element, so the computation time is the same (one for determining if the object must be created, and the other one for throwing the exception).